src/style.css

```diff
@@ -0,0 +1,90 @@
+.product-card {
+  box-sizing: border-box;
+  display: flex;
+  flex-flow: column nowrap;
+  width: 200px;
+  height: 403px;
+  background: #fff;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  font-family: Roboto;
+  font-weight: normal;
+  font-size: 11px;
+  line-height: 13px;
+  padding: 14px 14px 15px;
+}
+
+.product-image {
+  width: 160px;
+  height: 134px;
```

Comment:
To better sizing, try to use only width or height.

Reply:
160/134 px isn't original size of image, its size that I need for this card, if I remove width property it gets bigger